### PR TITLE
[zh] Sync authorization/authz-ingress/index.md

### PR DESCRIPTION
**Pull request by windsonsea**
Update after #14812

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/docs/tasks/security/authorization/authz-ingress/index.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ test: yes
 
 {{< tabset category-name="config-api" >}}
 
-{{< tab name="Istio APIs" category-value="istio-apis" >}}
+{{< tab name="Istio API" category-value="istio-apis" >}}
```
Comment (hanxiaop) [inline content/zh/docs/tasks/security/authorization/authz-ingress/index.md:34]: Maybe the revision of tab names are incorrect? I saw only `Istio APIs` and `Gateway API` in the original doc.

Comment (windsonsea) [reply content/zh/docs/tasks/security/authorization/authz-ingress/index.md:34]: Only removed the plural `s`, which has a different style in zh. Maybe we can use `相关 Istio API` in Chinese to show the plural APIs?

Comment (hanxiaop) [reply content/zh/docs/tasks/security/authorization/authz-ingress/index.md:34]: I think we have the `Istio APIs` tab usage among other Chinese/English pages, and this change seems to be inconsistent?

Comment (windsonsea) [reply content/zh/docs/tasks/security/authorization/authz-ingress/index.md:34]: Ok, all right, we can revert this change in this PR

Comment (windsonsea) [reply content/zh/docs/tasks/security/authorization/authz-ingress/index.md]: Well, all are reverted from `{{< tab name="Istio API"` to `{{< tab name="Istio APIs"` now

**Review by hanxiaop — APPROVED** [review]

**Review by wilsonwu — APPROVED** [review]
LGTM

**Pull request merged**